cirq/protocols/json_test_data/TrialResult.json

```diff
@@ -0,0 +1,297 @@
+[
+  {
+    "cirq_type": "TrialResult",
+    "params": {
+      "cirq_type": "ParamResolver",
+      "param_dict": [
+        [
+          {
+            "cirq_type": "sympy.Symbol",
+            "name": "a"
+          },
+          0.5
+        ]
+      ]
+    },
+    "measurements": {
+      "m": {
+        "packed_digits": "d32a",
```

Comment:
why doesn't this have a `cirq_type`

Reply:
Talked offline and decided this is unnecessary.